### DISCUSSION TITLE
bundle-analyzer-plugin should output stats.json

### DISCUSSION THREAD

**Discussion by jpvincent** [vercel/next.js#46242] [Ideas]
### Describe the feature you'd like to request

stats.json is essential when analyzing the size of a bundle. In complement to the sizes of the modules, it allows to know why a module has been required. 
If having it by default to true looks be opiniated : at least we could pass the option to the BundleAnalyzerPlugin

### Describe the solution you'd like

We could either have a parameter that would be passed directly to the plugin, allowing any options accepted by the plugin.
Or, like the current `openAnalyzer = true`  default value, we could just have the `generateStatsFile` given, as `true` (my preference : too few people know this file, that would incentive the curious to look at it).

I made a draft PR : https://github.com/vercel/next.js/pull/46233 

### Describe alternatives you've considered

Since a few years, on every Next project I have to optimize, I modify the `node_modules/@next/bundle-analyzer/index.js` directly locally to enable this option.

**Comment by backflip** [comment]
I would very much appreciate adding this option.
